printer_mods/chilicoke/Low_profile_exhaust_duct_adapter/README.md

```diff
@@ -0,0 +1,17 @@
+# Low Profile Exhaust Duct Adapter
+
+This adapter allows you to fit 4" flexable dryer duct to your 120mm exhaust fan with minimal rear protrusion. It is quick release held on with magnets.
+
+![Completed pic](installed.jpg?raw=true)
+
+Part needs to be assumbled with super glue, also requires minimum 8 6x4mm disk magnets for the quick release. Multiple thinner magents can also be used if stacked together for additional pull strength.
+Holes for magnets should be a press fit if printed with .4mm nozzle, I printed mine with .8 and they needed to be reamed for a good fit. 
+
+You will also need to replace the 4x fan screws with longer M3x35 SHCS. 
+
+![apply glue to the highlighted surfaces](glue_surfaces.png?raw=true)
+
+I was able to print everything standing upright without supports with 0.8mm Volcano, some steep overhangs on rear body will be ugly, but those parts can be trimmed off since it is a glue surface and not visible when assembled. 
+![preferred orientation](print orientation.png)
```

Comment:
Because there is a space in the file-name, print orientation.png does not render in the readme. Please fix the filename and the associated link in the readme to make it display properly.

Reply:
Appreciate your time and suggestions, all STLs had been improved to avoid excessive overhangs and should be much easily printable.